examples/pitr/variables.tf

```diff
@@ -45,6 +45,12 @@ variable "pitr_id" {
   description = "The ID of the source deployment PostgreSQL instance that you want to recover back to. The PostgreSQL instance is expected to be in an up and in running state."
 }
 
+variable "pitr_time" {
+  type        = string
+  description = "The timestamp in UTC format (%Y-%m-%dT%H:%M:%SZ) for any time in the last 7 days that you want to restore to. If no value is passed, earliest_point_in_time_recovery_time will be used as pitr_time. To retrieve the timestamp, run the command (ibmcloud cdb postgresql earliest-pitr-timestamp <deployment name or CRN>). For more info on Point-in-time Recovery, see https://cloud.ibm.com/docs/databases-for-postgresql?topic=databases-for-postgresql-pitr"
+}
```

Comment:
Given the background reading required to understand this issue, I feel that something should go in the example.

We can not have a required variable that says `if no value is passed`.
The descriptions says it will use the `earliest_point_in_time_recovery_time`, then says how to look it up. 

So I was thinking a description along the lines:

    description = "The timestamp in UTC format (%Y-%m-%dT%H:%M:%SZ) for any time in the last 7 days
        that you want to restore to. Pass an empty string to use the `earliest_point_in_time_recovery_time`.
        For more info on Point-in-time Recovery, see 
        https://cloud.ibm.com/docs/databases-for-postgresql?topic=databases-for-postgresql-pitr. There
        is no special value for the `latest_point_in_time_recovery_time`."
        
I am happy, for the example, that this is a require variable, that seems a natural fit. The default in the main module is null, so that is good as well.

Reply:
Done.